cl-base.obo

```diff
@@ -23306,11 +24477,9 @@ relationship: BFO:0000050 UBERON:0002367
 id: CL:1000301
 name: fibroblast of subepithelial connective tissue of prostatic gland
 def: "A fibroblast that is part of the subepithelial connective tissue of prostatic gland." [GOC:tfm]
-subset: human_reference_atlas
```

Comment:
@anitacaron, I believe you commented on edits to the human_reference_atlas before. I don't interact with this subset... do you know where these changes originate?

Reply:
Yes, these changes are linked to the changes on the ASCT+B tables. No need to worry about them.